content/en/docs/ops/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -72,6 +72,12 @@ $ helm upgrade istiod istio/istiod -n istio-system
 
 The ztunnel DaemonSet is the L4 node-proxy component of ambient.
 
+{{< warning >}}
+As ambient is not yet stable, the following statement is not a compatibility guarantee and is subject to change, or removal.
```

Comment:
```suggestion
As ambient is not yet stable, the following statement is not a compatibility guarantee and is subject to change, or removal. During alpha stage, ztunnel and/or the control plane may receive breaking changes that prevent compatibility between minor versions.
```

I'd probably add a note like this to specifically spell out the breaking changes.

Reply:
Yeah I wanted to not mention `alpha` so that we wouldn't have to update this part when switching to beta, since, strictly speaking, we are able to change this during beta as well.

Reply:
True. I don't think it's a huge ask to remove this during beta, but I also see the ambient-alpha-warning boilerplate so it may not be necessary now that I think more on it as we're already stating that at the top of the document.

Reply:
Anyway I used something like this that I think captures the gist, LMK